tests/CourseDashboardTests.cpp

```diff
@@ -1,8 +1,35 @@
 #include <gtest/gtest.h>
 #include "CourseDashboard.hpp"
+#include "User.hpp"
 
+class CourseDashboardTests : public ::testing::Test
+{
+protected:
+    CourseDashboard* testCdb;
+    User* testU1;
+
+    virtual void SetUp()
+    {
+        testU1 = new User("Szymon", "SzymonGajewski", "wieczorowa", "SzymonGajewski", "SzymonGajewski", "mail@com", "password");
+        testCdb = new CourseDashboard();
+    }
+
+    CourseDashboardTests()
+    {
+        EXPECT_NE("mail", testU1->getEmail());
+        EXPECT_EQ("mail@com", testU1->getEmail());
+        EXPECT_TRUE(testCdb->logIn());
```

Comment:
how is user input tested? logIn() uses cin.

Reply:
All EXPECTs swapped to ASSERT_TRUE.

Reply:
New class has been created to derive logIn function from CourseDashboard. In test function we are able to set password and login. After that we can use redefined logIn() function without any cin.